core/src/Xios.cpp

```diff
@@ -1412,7 +1366,8 @@ void Xios::createFile(const std::string fileId)
         // TODO: Account for diagnostics (#917)
     }
     if (periodStr.length() == 0 || periodStr == "0") {
-        setFileOutputFreq(fileId, stopTime - startTime);
+        ModelMetadata& metadata = ModelMetadata::getInstance();
+        setFileOutputFreq(fileId, metadata.stopTime() - metadata.startTime());
```

Comment:
Isn't `metadata.runLength()` exactly `metadata.stopTime() - metadata.startTime()`? (It should be)

Reply:
Good point, thanks @timspainNERSC. Addressed in 326c552e.